app/javascript/components/subject/MonitoringStatus.js

```diff
@@ -109,8 +109,9 @@ class MonitoringStatus extends React.Component {
         this.setState({
           showPublicHealthActionModal: true,
           message: `latest public health action to "${event.target.value}"`,
-          message_warning:
-            'The monitoree will be moved to the "Records Requiring Review" line list if they meet a recovery definition or will remain on the "Reporting" or "Non-Reporting" line list as appropriate until a recovery definition is met.',
+          message_warning: 'This will not impact the line list on which this record appears.',
+          household_warning:
+            'If any household members are being monitored in the exposure workflow, those records will appear on the PUI line list if any public health action other than "None" is selected above. If any household members are being monitored in the isolation workflow, this update will not impact the line list on which those records appear.',
```

Comment:
I'm assuming all this language was obtained from the product team since I don't see it on the ticket?

Reply:
correct